src/env_loader.zig

```diff
@@ -85,7 +94,12 @@ pub const Lexer = struct {
 
                     try writer.writeAll(variable.value[last_flush .. start - 1]);
                     last_flush = i;
-                    const name = variable.value[start..i];
+                    var end: usize = i;
+                    if (with_curly_braces) {
```

Comment:
`const` is faster than `var` and we should use it when possible 

We can avoid the branch here:
```suggestion
  const end  = i - @as(usize, @boolToInt(with_curly_braces));
  start  += @as(usize, @boolToInt(with_curly_braces));
```